client/profile-wizard/steps/business-details/flows/selective-bundle/selective-extensions-bundle/index.js

```diff
@@ -328,6 +328,34 @@ const getVisiblePlugins = ( plugins, country, industry, productTypes ) => {
 	);
 };
 
+/**
+ * Returns bundles that have at least 1 visible plugin.
+ *
+ * @param {Array} bundles  list of bundles
+ * @param {string} country  Woo store country
+ * @param {Array} industry List of selected industries
+ * @param {Array} productTypes List of selected product types
+ *
+ * @return {Array} Array of visible bundles
+ */
+const getVisibleBundles = ( bundles, country, industry, productTypes ) => {
```

Comment:
Naming here can be a bit questionable, but I'm trying to distinguish between a bundle (collection of plugins) and plugins since the current term "extensions" can be used interchangeably with "plugins" I think.